src/streaming.jl

```diff
@@ -215,7 +215,7 @@ function parsefile(filename::AbstractString,callbacks::XPCallbacks; bufferlines=
                 write(io, readline(file))
                 i += 1
             end
-            txt = Compat.bytestring(io)
+            txt = Compat.String(io)
```

Comment:
ah right, perhaps readstring or similar would work better. either way, dropping 0.3 support is fine.